test/src/gaussianStressTest.cpp

```diff
@@ -0,0 +1,146 @@
+#include <gtest/gtest.h>
```

Comment:
Ok, why U called them Stress test?

Reply:
This test runs gaussian noise over time (more than one iteration) and checks whether the values are the same. In the previous implementation, we had a bug in that the point cloud expanded (gaussian noise had been applied on the last noised point cloud).

Reply:
Ok, so this is a large portion of repetition of the unit test, not the stress test.

Reply:
Resolved offline.